components/drafts/index.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {getCurrentUser, getStatusForUserId} from 'mattermost-redux/selectors/entities/users';
+import {getTeammateNameDisplaySetting} from 'mattermost-redux/selectors/entities/preferences';
+import {displayUsername} from 'mattermost-redux/utils/user_utils';
+
+import {GlobalState} from 'types/store';
+
+import {makeGetDrafts} from 'selectors/drafts';
+
+import Drafts from './drafts';
+
+function makeMapStateToProps() {
+    const getDrafts = makeGetDrafts();
+    return (state: GlobalState) => {
+        const user = getCurrentUser(state);
+        const status = getStatusForUserId(state, user.id);
+
+        return {
+            displayName: displayUsername(user, getTeammateNameDisplaySetting(state)),
```

Comment:
It's because both channel and thread drafts needed this prop to render on the header,
hm, now that I said header maybe this belongs there.
ah, it's getting used in draft_actions as well, I think, let me check.

Reply:
Ah, I see where it is in the design. That looks much better than I would've expected